atlas/__init__.py

```diff
@@ -1,2 +1,3 @@
+from atlas.exceptions import ExceptionAsContinue
```

Comment:
I think we should leave ExceptionAsContinue in `atlas.exceptions` rather than make it top-level.